CHANGELOG.md

```diff
@@ -18,6 +19,44 @@ drc restart resource cache
 drc up -d enrich-submission
 ```
 
+#### Add second op-consumer
+
+```
+drc down
+```
+
+Update the `docker-compose.override.yml`:
+```
+  op-consumer-for-auth:
```

Comment:
We should also update the name of the other consumer in the override when deploying